Azure/sonic-buildimage
# Convert docker-fpm-frr to buster

Signed-off-by: Joyas Joseph <joyas_joseph@dell.com>

**- Why I did it**
This is an enhancement.

**- How I did it**
Code change

**- How to verify it**
Build `target/docker-fpm-frr.gz`
Verify that `/etc/apt/sources.list` points to `buster` using ` docker exec bgp cat /etc/apt/sources.list`

BGP neighborship  is established.
```
root@sonic:~# show ip bgp summary 

IPv4 Unicast Summary:
BGP router identifier 10.1.0.1, local AS number 65100 vrf-id 0
BGP table version 1
RIB entries 1, using 184 bytes of memory
Peers 1, using 20 KiB of memory

Neighbor        V         AS MsgRcvd MsgSent   TblVer  InQ OutQ  Up/Down State/PfxRcd
6.1.1.1         4        100      96      96        0    0    0 01:32:04            0

Total number of neighbors 1
root@sonic:~#  

```

**- Description for the changelog**
Convert docker-fpm-frr to buster


**- A picture of a cute animal (not mandatory but encouraged)**
